lib/switchtoactivity.js

```diff
@@ -0,0 +1,38 @@
+var waitForActivity = require('./waitforapp').waitForActivity,
+    getApp = require('./getapp').getApp;
+
+/**
+ * Switch focus to an app that is run as inline activity based on the origin.
+ * @param {Apps} apps to manage state.
+ * @param {String} origin of the application.
+ * @param {String} [entrypoint] of the application.
+ * @param {Function} callback [Error err].
+ */
+function switchToActivity(apps, origin, entrypoint, callback) {
```

Comment:
I see you use file-per-public_method so that is why it's in a separate file. It led to some code duplication between ```switchtoapp.js``` and ```switchtoactivity.js``` files, maybe some common parts can be moved to a separate private module, but tried to keep PR small for now.

AFAIK we can have only one inline activity per app origin at time (not sure about circular activities though), so it seems safe enough to not have dedicated parameters that describe parent app origin (like "switch to Messages app that is run as inline activity inside Contacts app" vs "switch to Messages app that is run as inline activity").

What do you think?